recipes/sregistry/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "sregistry" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3092be0059ae837be1cc69e4943610e729eb3facf49f1fed05a37a75349adb48
+
+build:
+  number: 0
+  entry_points:
+    - sregistry=sregistry.client:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - setuptools
+  run:
+    - awscli >=1.16.19
+    - boto3 >=1.7.83
+    - dropbox >=8.5.1
+    - globus-sdk >=1.5.0
+    - google-api-python-client >=1.6.4
+    - google-cloud-storage >=1.4.0
+    - oauth2client >=3.0
+    - pygments >=2.1.3
+    - python >=3
+    - requests >=2.18.4
+    - requests-toolbelt >=0.8.0
+    - retrying >=1.3.3
+    - spython
+    - sqlalchemy
+
+test:
+  imports:
+    - sregistry
+    - sregistry.auth
+    - sregistry.client
+    - sregistry.database
+    - sregistry.logger
+    - sregistry.main
+    - sregistry.main.__template__
```

Comment:
Do you think it' necessary to include __template__ ? It's just a dummy folder to show an example for starting a client. I'd say it's okay to leave out.

Reply:
Done